llvm/test/CodeGen/RISCV/GlobalISel/irtranslator/vec-ld.ll

```diff
@@ -880,3 +880,25 @@ define <vscale x 2 x i64>  @vload_nx2i64_align32(ptr %pa) {
   ret <vscale x 2 x i64> %va
 }
 
+define <vscale x 8 x i64*>  @vload_nx8i64_ptrs(ptr %pa) {
```

Comment:
`vscale x 8 x ptr`. You can probably add it to the same test file as all the other ones.